src/Symfony/Component/ClassLoader/ApcClassLoader.php

```diff
@@ -67,7 +67,7 @@ class ApcClassLoader
      */
     public function __construct($prefix, $decorated)
     {
-        if (!extension_loaded('apc')) {
+        if (!ini_get('apc.enabled') || ('cli' === PHP_SAPI && !ini_get('apc.enable_cli'))) {
```

Comment:
this could be a potential bc break for people who installed apc but didn't enable it. And I don't feel like it's the classes responsibility to check if apc is enabled. it should only make sure the methods can be called.